en/chronos.rst

```diff
@@ -203,20 +219,48 @@ objects::
     echo $date;
 
     // Different standard formats
-    echo $date->toAtomString();
-    echo $date->toCookieString();
-    echo $date->toIso8601String();
-    echo $date->toRfc822String();
-    echo $date->toRfc850String();
-    echo $date->toRfc1036String();
-    echo $date->toRfc1123String();
-    echo $date->toRfc2822String();
+    echo $dt->toAtomString();      // 1975-12-25T14:15:16-05:00
```

Comment:
@antograssiot decided to use `$time` since a plain date object would not make a lot of sense there